Hyperlint Automation: Meta Description Fixes (#20159)
cloudflare/cloudflare-docs: src/content/docs/fundamentals/setup/account/account-security/multi-factor-email-authentication.mdx

---
title: Multi-Factor Email Authentication
pcx_content_type: troubleshooting

> description: Enhance account security with Cloudflare's Multi-Factor Email Authentication. Prevent unauthorized access with one-time codes sent to your email.

Reviewer: ```suggestion
description: Cloudflare's Multi-Factor Email Authentication prevents unauthorized access by sending one-time codes to your email.
```
